.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 python:
   - "3.8"
 
-install: "pip install . pre-commit"
+install: "pip install Cython && pip install . pre-commit"
```

Comment:
Could this be because install_requires happens only at install time, but cython basically needs to be a dev requirement?  Should we make a requirements-dev.txt file which has cython and nose in it?  That's what I have done in other projects.  Then the first half becomes pip install -r `requirements-dev.txt`

Reply:
Applied in the new PR.